integration/test_collection.py

```diff
@@ -60,6 +61,15 @@
 DATE3 = datetime.datetime.strptime("2019-06-10", "%Y-%m-%d").replace(tzinfo=datetime.timezone.utc)
 
 
+def get_numpy_vector(input_list: list) -> Any:
```

Comment:
we have numpy in our dev requirements, so you can assume it is present for tests

Reply:
thanks, simplified